.github/workflows/pf2e-font.yml

```diff
@@ -0,0 +1,29 @@
+name: 'pf2e-font'
+on:
+  workflow_call:
+permissions:
+  contents: read
+  pull-requests: read
+jobs:
+  check-ttf:
+    name: Check reproducibility of generating PF2ESymbols.ttf
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install fontforge
+      run: sudo apt-get install fontforge-nox
+    - name: Generate PF2ESymbols.ttf
+      working-directory: fonts/PF2ESymbols
+      run: mkdir build && fontforge -lang=ff -c 'Open("pf2e-symbols.sfd"); Generate("build/PF2ESymbols.ttf")' 
+      id: generate-ttf
+    - name: Check equality of generated ttf file with checked-in ttf file
```

Comment:
Are they stable, or does the TTF contain information like a timestamp when it was generated or something like that?

Reply:
I'm not really sure. I first thought they would be somewhat stable, as regenerating with the same settings lead to the same binary; but I reran the workflow here after some more time and there were some changes now. (after a day-break so maybe there's a day but no timestamp or something like that). Also in the generation UI in fontforge there's a checkbox that can be ticked to include the timestamp, so I guessed that there would be no timestamp when not checking that.

I don't really know, I'd like it if we could could configure the generation to be reproducible, but I'm not sure how feasible that is.

Reply:
> to include the timestamp

Isn't that to prepend the timestamp to the file name?

There is a `UniqueID` added during export it seems, which contains the date…
<img width="769" alt="image" src="https://github.com/Toxaris/pathfinder2-cards/assets/1448874/946013e5-4a94-4f73-8255-c79813c13236">

Reply:
Oh I see then (at least) that part is not reproducible.